8103314 started failing a test

I just installed Gooey, and I had a failing test on current master. I nailed the cause down with git bisect to the commit 81033147956716981321da89b968a40345934e72. 
The (only) failing test is:
```
============================================== FAILURES ==============================================
___________________________________________ test_mutually ____________________________________________

exclusive_group = [_StoreTrueAction(option_strings=['-i'], dest='option1', nargs=0, const=True, default=False, type=None, choices=None, ...strings=['-j'], dest='option2', nargs=0, const=True, default=False, type=None, choices=None, help='hhh', metavar=None)]

    def test_mutually(exclusive_group):
      target_arg = find_arg_by_option(exclusive_group, '-i')
>     json_result = build_radio_group(exclusive_group)[0]
E     KeyError: 0

gooey/tests/argparse_to_json_unittest.py:151: KeyError
````
which makes sense seeing as 81033147956716981321da89b968a40345934e72 changed the return arg of `build_radio_group`.
I'm on Python 2.7.10 with Anaconda on Linux.

Pinging @chriskiehl

--- a/gooey/tests/argparse_to_json_unittest.py
+++ b/gooey/tests/argparse_to_json_unittest.py
@@ -148,7 +148,7 @@ def test_is_counter(empty_parser):
 
 def test_mutually(exclusive_group):
   target_arg = find_arg_by_option(exclusive_group, '-i')
-  json_result = build_radio_group(exclusive_group)[0]
+  json_result = build_radio_group(exclusive_group)
 
   data = json_result['data'][0]
   assert 'RadioGroup' == json_result['type']

